controlsfx/src/main/java/impl/org/controlsfx/spreadsheet/GridVirtualFlow.java

```diff
@@ -171,15 +171,8 @@ public void changed(ObservableValue<? extends Number> observable, Number oldValu
         });
     }
 
-//    @Override
-//    public void show(int index) {
-//        super.show(index);
-//        layoutTotal();
-//        layoutFixedRows();
-//    }
-
     @Override
-    public void scrollTo(int index) {
+    public void scrollToTop(int index) {
```

Comment:
Why is this getting renamed to `scrollToTop` ? Can't the developer enter an index value which scrolls it in the other direction?

Reply:
I override this method so that when a row is scrolled to, at the top (therefore should be the first one at the top), if we have some fixed rows, it will still be visible.

I thought that it was the VirtualFlow `scrollTo()` that was called when the TableView `scrollTo()` method was called. But apparently it's "scrollToTop", therefore I changed it. I'm unsure yet when VirtualFlow "scrollTo" is called and what it actually does. I'm currently manipulating a lot the SpreadsheetView and discovering issues along the way.